Miscellaneous/scf-dictionary

```diff
@@ -0,0 +1,49 @@
+import pickle
+
+from wrap_pyscf import *
+
+scf_guess = {}
+
+filename = "scf_record"
+
+#with open(filename, 'rb') as outfile:
+    #scf_guess = pickle.load(outfile)
+
+if "EthaneA 1Cx2" not in scf_guess:
```

Comment:
I'm going to declare the coordinate aspect of this code to possibly have some error; since all I have from avogadro is a straight up list of cartesian coordinates for atoms, I have to give my best judgement on deciding what hydrogens belong to what carbon. Since I started the alkane chain as having 0 for the  y and z components of the carbons, I'm grouping atoms by proximity along the x axis, and making sure that the y and z components are not far off from each other as well.

Reply:
I was wondering how you were doing that. Take a look at the [label feature](https://avogadro.cc/docs/display-types/) of Avogadro; it'll tell you which atom is which.